svelte/src/routes/v2/scripts/filter/lift-edges.ts

```diff
@@ -1,17 +1,18 @@
 import type {
 	ConvertedNode,
 	NodesDictionaryType,
-	PreGraphData,
 	ConfigInterface,
-	PreGraphDataEdge,
+	GraphDataEdge,
+	GraphData,
+	GraphDataNode
 } from '../../types';
 
 export interface FilteredNode extends ConvertedNode {
 	parent?: FilteredNode;
 }
-
+// HELPER FUNCTIONS
 // Find length of common prefix of 2 string arrays
```

Comment:
I think the name of the function and description should be changed before it turns into confusion

Reply:
will do, wait

Reply:
done